### ISSUE TITLE
blobstore does not work on gae

### ISSUE THREAD

**Issue by billy1380** [manolo/gwtupload#10]
The blobstore uploader does not work correctly after being deployed on app engine. For the record it works well on the test server.

The url returned by the upload service is empty/null/undefined. But when I check on the blobstore viewer the image has definitely been uploaded.

Been testing on Firefox and Chrome. Chrome seems to fail silently with not even a console message where as Firefox shows a message with a warning about the blob store not being usable without billing (which I am pretty sure is not true anyway - especially since the files are on the blobstore).

The now deprecated filestore api works quite well and the files can be seen in the blobstore as expected.

**Comment by slugmandrew** [comment]
What's happened to this project? I'm a bit hesitant to start using it tbh seeing as the docs are out-of-date and there are PRs still outstanding

**Comment by billy1380** [comment]
Not heard from the owner about pull request 13 which tries to "fix" this issue by using google cloud storage instead of the blob store. But I have been using that version on a couple of projects and it works well.

**Comment by slugmandrew** [comment]
Hmm. Any idea what was wrong? I use the blobstore now and it works really well in production. As far as I know there are no plans to deprecate it.

**Comment by billy1380** [comment]
Yeah... you are right... it is the blobstore file api (that allowed you to write to blobs as streams) that has been deprecated and now removed.
